miscellaneous_scripts/dockerfile_patch_script.sh

```diff
@@ -29,6 +31,20 @@ mv $PATCHING_INFO_PATH/patch-details-current $PATCHING_INFO_PATH/patch-details
 chmod +x $PATCHING_INFO_PATH/patch-details/install_script_language.sh && \
 $PATCHING_INFO_PATH/patch-details/install_script_language.sh
 
+if [ $LATEST_RELEASED_IMAGE_URI == "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:2.0.1-gpu-py310-cu121-ubuntu20.04-sagemaker" ]; then
```

Comment:
Is there a way to do this without specific conditional logic, i.e. providing the SMP_URL in the core_packages.json and parsing this way? This precedent may result in this conditional logic expanding and becoming more difficult to maintain

Reply:
Yes, this is a makeshift strategy for the time being. We will expand this logic to prevent these conditionals to exist.